README.md

```diff
@@ -76,7 +76,7 @@ Other features of ArangoDB include:
   durability or more performance
 - **Schema-free schemata** let you combine the space efficiency of MySQL with the
   performance power of NoSQL
-- Free **index choice**: use the correct index for your problem, be it a skiplist 
+- Free **index choice**: use the correct index for your problem, be it a persistent 
```

Comment:
```suggestion
- Free **index choice**: use the correct index for your problem, be it a persistent index
```

Reply:
I also think this is not a good sentence anymore: previously we had different index types such as hash, skiplist, fulltext, geo etc.
Now we basically have the generic index plus geo and fulltext.

So we should probably turn this into something like
"ArangoDB supports various index types to support different use cases, e.g. graph queries, geo location queries, fulltext searches